katpoint/projection.py

```diff
@@ -391,24 +566,15 @@ def sphere_to_plane_arc(az0, el0, az, el):
 
     Raises
     ------
-    ValueError
-        If input values are out of range
-
+    OutOfRangeError
+        If an elevation is out of range and out-of-range treatment is 'raise'
     """
     ortho_x, ortho_y, cos_theta = sphere_to_ortho(az0, el0, az, el)
     # Safeguard the arccos, as over-ranging happens occasionally due to round-off error
     theta = np.arccos(np.clip(cos_theta, -1.0, 1.0))
```

Comment:
I like it. While ARC is the main use case, there could also be issues at the upper end (ie points near the origin of the plane). When `min_cos_theta` is provided, the lower end is safe, but if then `cos_theta > 1.0` it will blow up `sin_theta = np.sqrt(1.0 - cos_theta * cos_theta)`.

Done.